contents/blog/data-management-feature.md

```diff
@@ -0,0 +1,101 @@
+
+![PostHog - data management tab](../images/blog/data-management-feature/data_management_tab.png)
+
+Data Management introduces more ways to organize and categorize your events. Event definitions can now be organized using tags and we've created the 'Verified' marker, which tells collaborators that an event should be favored over other similar events. 
+
+Verified events are also prioritized within filters and other selection components, and we've introduced to two sortable "freshness" indicators (30-day volume and query volume) to make it easier to choose the correct events when creating Insights.
+
+As part of this overhaul, we've made a few important changes to how data is organized, too. 'Event Stats' and 'Property Stats' have been renamed 'Events' and 'Event properties', and we've moved them into the Data Management tab alongside 'Actions'. The 'Events & Actions' tab has been renamed 'Live Events', where you can debug incoming event data by streaming it in near real time.
+
+![PostHog - live events tab](../images/blog/data-management-feature/live_events.png)
+
+## Making it easy to define events 'on the fly'
+
+Not only are we giving you more ways to organize and define events, we're also making it easy to edit them wherever you are in the product.
+
+Building an insight and see an Event or Event property that could be more descriptive? You can now create a definition for that without leaving the insight query builder.
+
+Just hover over the Event property in the dropdown...
+
+![PostHog - read definition hover card](../images/blog/data-management-feature/read_definition_card.png)
+
+And when you click 'Edit' you can add a description, add tags or mark it as a Verified Event.
+
+![PostHog - write definition hover card](../images/blog/data-management-feature/write_definition_card.png)
+
+Alternatively, click on 'View' in the definition hover card and you'll be redirected to the new Data Management page, where you can explore all the Events and Event properties you've created.
+
+![PostHog - view definition context](../images/blog/data-management-feature/data_management_view.gif)
+
+And if you want a gold star in best practice, you can also opt to create definitions with our API. Check out our documentation on how to use the [`event_definitions`](/docs/api/event-definitions) and [`property_definitions`](/docs/api/property-definitions) API endpoints.
+
+Oh, and in case you were wondering, the awesome thing about definitions in PostHog is that they can be created, updated, or deleted at any time without affecting the event data that is already there, so **you don't need to create event definitions before ingesting events**. 
+
+## How to use Data Management as a team (best practices)
+
+To help you in your journey to data management nirvana, here are a few rules of thumb to consider when using these new features.
+
+1. The more definitions you add, the easier it will be for your teammates to understand what each event and property represents.
```

Comment:
```suggestion
**1. The more definitions you add, the easier it will be for your teammates to understand what each event and property represents.**
```